### ISSUE TITLE
The toast pops up under a modal in bootstap 5. It gets overlapped by the modal

### ISSUE THREAD

**Issue by StilinskiCyril** [ankurk91/vue-toast-notification#88]
**I'm submitting a ...**  (check one with "x")

```
[ ] Bug report => search github for a similar issue or PR before submitting
[x] Feature request
[ ] Other, please describe
```

**Tell about your platform**

* Vue.js version : 3.2.x
* Browser name and version : Firefox
* This package version : 3.x

**Current behavior**
The toast pops up under a modal in bootstap 5. It gets overlapped by the modal

**Expected behavior**
The toast should overlap the modal

**Comment by ankurk91** [comment]
okay i will check it soon

**Comment by ankurk91** [comment]
For now, You can adjust the z-index of the toast element.

```css 
.v-toast {
  z-index: 9999 ! important
}
```

https://jsfiddle.net/91xjpa0o/

I will fix this in next version

**Comment by StilinskiCyril** [comment]
alright. Thanks

**Comment by ankurk91** [comment]
https://stackoverflow.com/questions/41815069/is-it-possible-to-position-a-div-on-top-of-a-dialog-tag-that-is-not-its-parent


The <dialog> element is added to the 'Top Layer' of the DOM which has its own stacking order (z-index does not affect this

**Comment by ankurk91** [comment]
fixed in 3.1.2
